sharq_project/__manifest__.py

```diff
@@ -13,28 +12,25 @@
     'author': "My Company",
     'website': "http://www.yourcompany.com",
 
-    # Categories can be used to filter modules in modules listing
-    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
-    # for the full list
     'category': 'Uncategorized',
     'version': '0.1',
 
-    # any module necessary for this one to work correctly
-    'depends': ['web', 'project'],
+    'depends': ['web', 'project','hr'],
 
-    # always loaded
     'data': [
         # 'security/ir.model.access.csv',
         'views/views.xml',
         'views/templates.xml',
         'views/project_view.xml',
```

Comment:
@Buniadi are these files needed in your project? If not needed then remove the unwanted xml files